dataset/variable_reduction.cpp

```diff
@@ -16,8 +16,7 @@
 namespace scipp::dataset {
 namespace {
 Variable applyMask(const VariableConstView &var, const Variable &masks) {
-  return scipp::variable::transform(var, masks,
-                                    scipp::core::element::convertMaskedToZero);
+  return scipp::variable::masked_to_zero(var, masks);
 }
```

Comment:
Why the trivial wrapper function?

Reply:
Must have misunderstood your previous comment. Were you suggesting moving the whole apply mask functionality to a common location instead?

Reply:
I think you understood correctly, I am just saying that you don't need to define `applyMask`, but just `masked_to_zero` directly?

Reply:
Good point will remove.